<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SleepReportMoreWindowBasic</class>
 <widget class="QDialog" name="SleepReportMoreWindowBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="SleepReportMoreWindowBasicBk">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>272</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>272</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="backSleepReportMoreButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>24</y>
      <width>38</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rsc/resource.qrc">
      <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="sleepReportMoreTitle">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>29</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#1788c3
</string>
    </property>
    <property name="text">
     <string>Sleep Report</string>
    </property>
   </widget>
   <widget class="QFrame" name="daysUsed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>45</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="daysUsedTitle">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>2</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BDBDBD</string>
     </property>
     <property name="text">
      <string>Days Used</string>
     </property>
    </widget>
    <widget class="QLabel" name="daysUsedText">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>0</y>
       <width>91</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2196F3</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daysUsedUnit">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>41</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#616161</string>
     </property>
     <property name="text">
      <string>Period</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="days4HrsPlus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>82</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="days4HrsPlusTitle">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>2</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BDBDBD</string>
     </property>
     <property name="text">
      <string>Days 4Hrs+</string>
     </property>
    </widget>
    <widget class="QLabel" name="days4HrsPlusText">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>0</y>
       <width>91</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2196F3</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="days4HrsPlusUnit">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>41</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#616161</string>
     </property>
     <property name="text">
      <string>Period</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="averageUsage">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>119</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="averageUsageTitle">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>2</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BDBDBD</string>
     </property>
     <property name="text">
      <string>Average Usage</string>
     </property>
    </widget>
    <widget class="QLabel" name="averageUsageText">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>0</y>
       <width>81</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2196F3</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="averageUsageUnit">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>41</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#616161</string>
     </property>
     <property name="text">
      <string>Hours</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="usedHours">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>156</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="usedHoursTitle">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>2</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BDBDBD</string>
     </property>
     <property name="text">
      <string>Used Hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="usedHoursText">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>0</y>
       <width>91</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2196F3</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="usedHoursUnit">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>41</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#616161</string>
     </property>
     <property name="text">
      <string>Hours</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="leak">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>193</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="leakTitle">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>2</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BDBDBD</string>
     </property>
     <property name="text">
      <string>Leak</string>
     </property>
    </widget>
    <widget class="QLabel" name="leakText">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>0</y>
       <width>101</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2196F3</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="leakUnit">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>20</y>
       <width>41</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#616161</string>
     </property>
     <property name="text">
      <string>L/min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ahi">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>231</y>
      <width>210</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3D3D3D;
border-radius:3px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ahiTitle">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>2</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BDBDBD</string>
     </property>
     <property name="text">
      <string>AHI</string>
     </property>
    </widget>
    <widget class="QLabel" name="ahiText">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>0</y>
       <width>171</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#2196F3</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="selectionPeriod">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>74</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD</string>
    </property>
    <property name="text">
     <string>Period</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendSleepReportButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>215</y>
      <width>118</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
    </property>
    <property name="text">
     <string>Send Sleep Report</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rsc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
